charts/linkerd-control-plane/README.md

```diff
@@ -230,10 +230,12 @@ Kubernetes: `>=1.21.0-0`
 | proxy.image.pullPolicy | string | imagePullPolicy | Pull policy for the proxy container Docker image |
 | proxy.image.version | string | linkerdVersion | Tag for the proxy container Docker image |
 | proxy.inboundConnectTimeout | string | `"100ms"` | Maximum time allowed for the proxy to establish an inbound TCP connection |
+| proxy.inboundDiscoveryCacheUnusedTimeout | string | `"90s"` | Maximum time allowed before an unused inbound discovery result is evicted from the cache, defaults to 90s if unspecified |
```

Comment:
Probably don't need to describe the default value in the comment since we can see the default value in the second column now.